README.md

```diff
@@ -83,128 +122,132 @@ Install with your preferred package manager:
+| `a`     | Enters append mode                                                              |
+| `e`     | Enters extend mode                                                              |
+| `c`     | Deletes the text inside selections and starts insert mode                       |
+| `n`     | `[count]` Finds the next match after the main selection                         |
+| `N`     | `[count]` Finds the previous match before the main selection                    |
+| `q`     | `[count]` Skips the current selection and finds the next one                    |
+| `Q`     | `[count]` Skips the current selection and finds the previous one                |
+| `]`     | `[count]` Swaps the main selection with next selection                          |
+| `[`     | `[count]` Swaps the main selection with previous selection                      |
+| `}`     | `[count]` Deletes the main selection and goes to next                           |
+| `{`     | `[count]` Deletes the main selection and goes to previous                       |
+| `j`     | `[count]` Creates a selection on the char below the cursor                      |
+| `J`     | `[count]` Skips the current selection and Creates a selection on the char below |
+| `k`     | `[count]` Creates a selection on the char above the cursor                      |
+| `K`     | `[count]` Skips the current selection and Creates a selection on the char above |
+| `p`     | Puts the text inside `unnamed register` before selections                       |
+| `P`     | Puts the text inside `unnamed register` after selections                        |
+| `y`     | Yanks the text inside selection to `unnamed register`                           |
+| `Y`     | Yanks the text from start of selection till end of line to `unnamed register`   |
+| `yy`    | Yanks the line of selection to `unnamed register`                               |
+| `z`     | Aligns selections by adding space before selections                             |
+| `Z`     | Aligns selections by adding space at beginning of line                          |
+| `d`     | Deletes the text inside selections                                              |
+| `D`     | `count` Deletes the text from start of selection till end of line               |
+| `dd`    | `count` Deletes line of selections                                              |
+| `@`     | Executes a macro at beginning of every selection                                |
+| `.`     | Reapets last change at the beginning of every selection                         |
+| `,`     | Clears All Selections except the main one                                       |
+| `:`     | Prompts for a normal command and Executes it at beginning of every selection    |
+| `u`     | Undo changes                                                                    |
+| `<C-r>` | Redo changes                                                                    |
```

Comment:
```suggestion
```
It's unnecessary cause at the end all of it are aligned and every time we want to add a mapping or edit some description we have to edit all of them again